pom.xml

```diff
@@ -1399,5 +1407,19 @@
         </pluginManagement>
       </build>
     </profile>
+    <profile>
```

Comment:
These convenience profiles are great for easy activation of specific groups/excludes, but do these need to be defined in Accumulo's pom? I'd think they'd be more appropriate in individual user's `settings.xml` files as a matter of individual convenience.